contents/customers/drivvn.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'How Drivvn drives growth with group analytics and feature flags'
+customer: Drivvn
+logo: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/octomind_logo_673e0ed777.png
+logoDark: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/octomind_logo_dark_a89deeee90.png
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_octomind_bb047603a6.jpg
+featuredCustomer: false
+industries:
+  - 'Automotive, Agency,SaaS'
+users:
+  - Marketing
+toolsUsed:
+  - Group Analytics
+  - Feature Flags
+  - Surveys
+  - Insights
+  - Web analytics
+date: 2024-11-10
+---
+
+What if shopping for a new vehicle was a user-centered, seamless, thoughtful, online experience — no dealership required? 
+
+That’s the future Drivvn has been building. Supporting a network of more than 60+ automotive ecommerce providers, Drivvn is the digital engine behind global vehicle brands from Ford to Fiat. Known for creating brand experiences that are design-driven, human-first and still do great at the “selling” part — you’d almost think it was magic.
+
+Under the hood, Drivvn’s Head of Product Engineering says it’s all PostHog. 
+
+John and his team have used PostHog to up their game for over two years. While his team designs and builds online shopping experiences for car brands, PostHog is what helps them optimize and customize the experiences for end users.
+
+To help you visualize it, as a user, let’s imagine you’re shopping for a Ford vehicle over on this site. Here, you’ll have an easy time comparing vehicles, and shopping for what you need — but what you may not know is that your experience is quickly being customized to better serve you, as you go along.
```

Comment:
Are we going to add in the box-out or highlight I suggested here? Up to you, but I think it could add a lot to help visualize Drivvn's concept.